### ISSUE TITLE
Quest shenanigans in Duskwood

### ISSUE THREAD

**Issue by Laonefaye** [tkrokli/TrinityCore#10]
I have a human paladin character I'm currently questing in Duskwood. So far, two quests seem to be having issues. The first, The Totem of Infliction (quest id = 101, given by Madame Eva, npc id = 265), has a problem with the Fetid Corpses (npc id = 1270) not dropping their ghoul fangs. I mowed through about 30 of them, and apparently they are all toothless. I was working on those particular ghouls because that is where the quest on the map was telling me to go. 
The second quest I'm having trouble with is The Night Watch (2) (quest id = 57; quest giver = 264, Commander Althea Ebonlocke), which requires the slaying of skeletal horrors and skeletal fiends. Well, I have no problem finding the horrors in the quest area, but the fiends (npc id = 531) are nowhere to be found, anywhere. I will likely just add them myself using gm commands, but there seem to be far more bugged quests and NPCs than I remember from the last incarnation of TrinityCore.

**Comment by tkrokli** [comment]
> and apparently they are all toothless.

While I work on release 1.4.1,
have fun comparing your findings with this video:

<a href="http://www.youtube.com/watch?feature=player_embedded&v=8lZlQNsbmeE" target="_blank"><img src="http://img.youtube.com/vi/8lZlQNsbmeE/0.jpg" alt="Quest for Tooth" width="240" height="180" border="10" /></a>

**Comment by Laonefaye** [comment]
LOL that looks about right. Apparently the drop rate for ghoul fangs in general is crap, but I remember it being a little better than 0 per 30-or-so. Maybe I should have killed 31? At least they're fun to kill, what with the Muppet sounds they make. Just been making a tally of things that are slightly weird that weren't weird when I played a year ago.

**Comment by tkrokli** [comment]
Yes, I know that drop rates have always been debated, even on retail (in comments on wowhead.com).
